test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ExportDataValidationTests.cs

```diff
@@ -357,5 +411,73 @@ private CloudStorageAccount GetCloudStorageAccountHelper(string storageAccountNa
 
             return cloudAccount;
         }
+
+        private async Task<(Dictionary<(string resourceType, string resourceId), string> serverData, string groupId)> CreateGroupWithPatient(bool includeAllCompartmentResources)
+        {
+            // Add data for test
```

Comment:
nit: another option would be add a json file with the sample data mentioned below and then read that in before sending to the fhir-server. We use this method in other places. The files are located in Microsoft.Health.Fhir.Tests.Common . I don't have a preference for either method. Just wanted to call that out as an option.

Reply:
For this test I think it works better to have it in the test file. I could do this with two JSON files and a batch, but I think it works better for the minimal resources this test needs to have it in the test file.